BUILD.bazel

```diff
@@ -23,14 +24,37 @@ pandoc_toolchain(
 )
 
 pandoc_binary(
-    name = "pandoc_binary"
+    name = "pandoc_binary",
+)
+
+config_setting(
+    name = "gapic_gen_python_3_6",
+    values = {"define": "gapic_gen_python=3.6"},
+)
+
+py_runtime(
+    name = "pyenv3_runtime",
+    interpreter = ":pyenv3wrapper.sh",
```

Comment:
Just to clarify, we're not spending O(minutes) building the correct python version, right?

Reply:
Yes, we are not building anything here, simply using the preinstalled one.